File: dataset/human/c/file_5609.c
#include<stdio.h>

int main(void){

    long int x,ans=0;

    scanf("%ld",&x);

    ans = (x / 11)*2;

    if(x % 11 == 0){
        ans = ans;
    }else if(x % 11 <= 6){
        ans += 1;
    }else if(x % 11 > 6){
        ans += 2;
    }

    printf("%ld",ans);
    return 0;
}

File: dataset/human/python/file_5609.py
from collections import deque
def main():

    s = deque(input())

    

    ans = 0
    while len(s)>1:
        if s[0]==s[-1]:
            s.popleft()
            s.pop()
            continue
        if s[0]!='x' and s[-1]!='x':
            ans = -1
            break
        if s[0]=='x':
            ans+=1
            s.popleft()
        if s[-1]=='x':
            ans+=1
            s.pop()
    
    print(ans)

if __name__ == "__main__":
    main()
